### PR TITLE
RTK Query in the new architecture

### PR DESCRIPTION
**Pull request by ekazankova**

### CODE REVIEW

**Comment by ipakhomov** [comment]
@ekazankova thank you for this update! At first please apply the following changes:
1. Get rid of [storeRef](https://github.com/RonasIT/react-native-starter/blob/ekazankova/rtkq-in-new-architecture/src/libs/shared/data-access/store/store-ref.ts).
2. Remove screen facades 
3. Replace `src/libs/shared/data-access/profile/store` and `service.ts`  with RTKQ API

**Comment by ekazankova** [comment]
@ipakhomov I will make profile RTKQ API in a separate branch with auth

File: src/app/scenes/app.tsx [hunk 1]
```diff
@@ -20,19 +22,23 @@ const setLanguage = useLanguage(
       ...require('@assets/i18n/en.json'),
```
Comment (ipakhomov) [inline src/app/scenes/app.tsx:22]: Please move this file into new directory `src/i18n/navigation`

File: src/app/scenes/app.tsx [hunk 2]
```diff
@@ -20,19 +22,23 @@ const setLanguage = useLanguage(
...
       ...require('../../libs/auth/assets/i18n/en.json'),
```
Comment (ipakhomov) [inline src/app/scenes/app.tsx:23]: As discussed, let's keep translation separately from assets and outside the `libs` directory. Please move  this file into `src/i18n/auth`.

Could you please apply the same change of other translation files?

File: src/tests/helpers/set-default-language.ts [hunk 3]
```diff
@@ -6,7 +6,8 @@ export function setDefaultLanguage(): object {
     ...require(`../../assets/i18n/${defaultLanguage}.json`),
```
Comment (ipakhomov) [inline src/tests/helpers/set-default-language.ts:6]: Please apply the same path adjustments as mentioned above

**Review by ipakhomov — CHANGES_REQUESTED** [review]
@ekazankova thanks for the update! Could you please also relocate the translation files as per our discussion?

File: src/app/scenes/app.tsx (hunk 1 above)
Comment (ekazankova) [reply src/app/scenes/app.tsx]: I decided to name this directory `scenes` because it can have translation not only for navigation, but also for screens

**Review by ipakhomov — APPROVED** [review]

**Pull request merged**